src/test/java/com/wikia/webdriver/testcases/mercurytests/old/PortableInfoboxTests.java

```diff
@@ -96,18 +94,16 @@ public void mercury_infobox_externalLinkRedirectsToCorrespondingUrl() {
     infobox.isExternalLinkLabelInURL(externalLinkName, externalURL);
   }
 
-  @Test(groups = "mercury_infobox_imagesAndVideosOpenInLightbox", enabled = false)
-  @RelatedIssue(issueID = "XW-1200", comment = "functionality is broken")
+  @Test(groups = "mercury_infobox_imagesAndVideosOpenInLightbox")
   public void mercury_infobox_imagesAndVideosOpenInLightbox() {
     init();
     navigate.toPage(MercurySubpages.INFOBOX_1);
 
     // Check image
     infobox
-        .clickHeroImage()
+        .clickOnImageInInfobox()
         .isLightboxOpened()
-        .closeLightbox()
-        .isTitleOverImageVisible();
```

Comment:
it looks like it was used as an indicator to check if lightbox is opened.
And because test should check that images and videos will be opened in lightbox this method is redundant